<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEditAsm"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="textBrowserBin"/>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="textBrowserHex"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonClear">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonByte">
        <property name="text">
         <string>Байт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonWord">
        <property name="text">
         <string>Слово</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
